pymedphys/_gui/streamlit/mudensity-compare.py

```diff
@@ -1271,7 +1288,7 @@ def run_calculation(
         )
 
     st.write("## Results")
-    st.pyplot()
+    st.pyplot(fig)
```

Comment:
note that the API from imagemagick has changed so any call to magick is at risk of failing if someone installs the newer version.  Suggest creating a command lookup that toggles based on which API is available.

Reply:
Yup, good call. Will fix.

Reply:
Yup, this is done now. See the implementation here:

https://github.com/pymedphys/pymedphys/blob/651d7aa7d7c4b25f884ce01bb32f80424a9a03eb/pymedphys/_gui/streamlit/mudensity/main.py#L956-L1005